students/pooria_k/lesson01/inventory_management/main.py

```diff
@@ -0,0 +1,81 @@
+"""Launches the user interface for the inventory management system"""
+from __future__ import print_function
+import sys
+from inventory_management import market_prices
+from inventory_management import inventory_class
+from inventory_management import furniture_class
+from inventory_management import electric_appliances_class
+
+
+def main_menu(user_prompt=None):
+    """Main Menu Method"""
+    valid_prompts = {"1": addnew_item,
+                     "2": item_info,
+                     "q": exit_program}
+    options = list(valid_prompts.keys())
+
+    while user_prompt not in valid_prompts:
+        options_str = ("{}" + (", {}") * (len(options)-1)).format(*options)
+        print("Please choose from the following options ({}):".format(options_str))
+        print("1. Add a new item to the inventory")
+        print("2. Get item information")
+        print("q. Quit")
+        user_prompt = input(">")
+    return valid_prompts.get(user_prompt)
+
+def get_price(item_code):
```

Comment:
Hi Pooria,

It seems this one was modified and the functionality of the function changed. It was originally like this:

```
def get_price(item_code):
    """Displays price of an item"""
    price = market_prices.get_latest_price(item_code)
    return price
```

Could you restore the function and update your unit testing accordingly?

Thanks,

Luis

Reply:
Hi Luis 
I got the file from 
https://github.com/UWPCE-PythonCert-ClassRepos/SP_Python220B_2019/blob/master/lessons/lesson01/assignment/inventory_management/main.py#L23

def getPrice(itemCode):
    print("Get price")

Is that the right repo to get the files for assignment ?